src/pages/workspaces/workspace/Data.js

```diff
@@ -16,7 +16,32 @@ import { Component } from 'src/libs/wrapped-components'
 import WorkspaceContainer from 'src/pages/workspaces/workspace/WorkspaceContainer'
 
 
-const filterState = _.pick(['pageNumber', 'itemsPerPage', 'selectedEntityType', 'selectedEntities'])
+const filterState = _.pick(['pageNumber', 'itemsPerPage', 'selectedDataType'])
```

Comment:
If you re-click on the same entity type, it's clearing the table without refreshing. We either need some additional data here, or we need to tweak the logic below, or both.

Reply:
Knew I'd miss something. Made it so that clicking the thing you're on is a no-op.

Reply:
Hmm, I'd think that should refresh the data. Is this connected to the fix I did recently?